android-gif-drawable/src/main/c/bitmap.c

```diff
@@ -55,7 +55,7 @@ void unlockPixels(JNIEnv *env, jobject jbitmap) {
 }
 
 __unused JNIEXPORT jlong JNICALL
-Java_pl_droidsonroids_gif_GifInfoHandle_renderFrame(JNIEnv *env, jclass __unused handleClass, jlong gifInfo, jobject jbitmap) {
+Java_pl_droidsonroids_gif_GifInfoHandle_00024Companion_renderFrame(JNIEnv *env, jclass __unused handleClass, jlong gifInfo, jobject jbitmap) {
```

Comment:
What is `00024` infix for?

Reply:
`GifInfoHandle` is not able to access this method post converting from java to kotlin without the infix

Reply:
OK, but what is the source of this numer value?

Reply:
Android Logcat had given me this suggestion while compiling.

Reply:
It does not sound convincing  😅 
We need to fully understand the actual name mangling rules.

Reply:
I agree. Let me see if I can get any info on this.

Reply:
I could find this: https://stackoverflow.com/a/49560732/5444161
Let me know if this helps

Reply:
Thanks, now it is clear.